### PR TITLE
Add UK2

### PR DESCRIPTION
**Pull request by LeeMcCartney**
2nd attempt. Hopefully I've got it right this time!

### CODE REVIEW

**Comment by Carlgo11** [comment]
Looks better but you forgot the spacing. Make sure your addition is aligned with other things.

**Comment by stephengroat** [comment]
@LeeMcCartney, do you have an `img` to add to the `img/domains` folder? it has to be a `png` with the size `32x32 px`

**Comment by LeeMcCartney** [comment]
I've uploaded the image to the directory but i'm not sure how to link it to my edit.

File: _data/domains.yml [hunk 1]
```diff
@@ -460,6 +460,13 @@ websites:
       tfa: No
       lang: de
 
+    - name: UK2
+      url: https://www.uk2.net/
+      twitter: UK2
+      facebook: uk2.net
+      img:
```
Comment (stephengroat) [inline _data/domains.yml]: could you add the filename `uk2.png` for the `img` tag?

also, rename the img to `uk2.png`? we try to only have the entry `name` in the `img` filename

it should look like this: `img: uk2.png`

**Review by stephengroat — CHANGES_REQUESTED** [review]

File: _data/domains.yml (hunk 1 above)
Comment (stephengroat) [reply _data/domains.yml]: bump for @LeeMcCartney

**Comment by LeeMcCartney** [comment]
My head hurts!

I can't work out how to rename the img file and I'm not sure what file i'm editing anymore.

**Comment by Carlgo11** [comment]
I'll do it in a couple of hours.

**Review by stephengroat — APPROVED** [review]

**Comment by stephengroat** [comment]
thanks for all of the work @LeeMcCartney! got the image renamed, corrected, etc.

File: _data/domains.yml [hunk 2]
```diff
@@ -460,6 +460,13 @@ websites:
       tfa: No
       lang: de
 
+    - name: UK2
+      url: https://www.uk2.net/
+      twitter: UK2
+      facebook: uk2.net
```
Comment (Carlgo11) [inline _data/domains.yml]: Please rename to `UK2.net`

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

**Review by Carlgo11 — APPROVED** [review]

**Pull request merged**